src/xdp/rx.c

```diff
@@ -954,6 +982,13 @@ XdpRxQueueCreate(
         goto Exit;
     }
 
+    if (HookId->Direction == XDP_HOOK_TX) {
+        DirectionString = L"_tx";
+    } else {
+        ASSERT(HookId->Direction == XDP_HOOK_RX);
+        DirectionString = L"";
+    }
```

Comment:
So only TX direction has a suffix?

Reply:
Yeah, RX queues are implicitly RX direction - that keeps our IDs relatively clean. Same for TX queues.